tests/e2e/specs/mouseposition.spec.js

```diff
@@ -20,32 +25,109 @@ function getMousePositionValue(x = 100, y = 100, coordStr = '671202, 6074296') {
 }
 
 describe('Test mouse position', () => {
-    beforeEach(() => {
-        cy.goToMapView('en', {
-            E: 2600000,
-            N: 1200000,
-            z: 8,
+    context('Position in footer', () => {
+        beforeEach(() => {
+            cy.goToMapView('en', {
+                E: 2600000,
+                N: 1200000,
+                z: 8,
+            })
+        })
+        it('Shows LV95 coordinates by default', () => {
+            getMousePositionValue(100, 100, '2435419.18, 1297425.86')
+        })
+        it('switches to LV03 when this SRS is selected in the UI', () => {
+            getMousePositionAndSelect(CoordinateSystems.LV03)
+            getMousePositionValue(100, 100, '435419.18, 297425.86')
+        })
+        it('switches to MGRS when this SRS is selected in the UI', () => {
+            getMousePositionAndSelect(CoordinateSystems.MGRS)
+            getMousePositionValue(100, 100, '31TFN 67815 97235 ')
+        })
+        it('switches to WebMercator when this SRS is selected in the UI', () => {
+            getMousePositionAndSelect(CoordinateSystems.WGS84)
+            getMousePositionValue(100, 100, '47° 48′ 23.71″ N 5° 14′ 28.92″ E')
+        })
+        it('goes back to LV95 display if selected again', () => {
+            // Change display projection without moving the mouse
+            getMousePositionAndSelect(CoordinateSystems.MGRS)
+            getMousePositionAndSelect(CoordinateSystems.LV95)
+            getMousePositionValue(100, 100, '2435419.18, 1297425.86')
         })
     })
-    it('Shows LV95 coordinates by default', () => {
-        getMousePositionValue(100, 100, '2435419.18, 1297425.86')
-    })
-    it('switches to LV03 when this SRS is selected in the UI', () => {
-        getMousePositionAndSelect(CoordinateSystems.LV03)
-        getMousePositionValue(100, 100, '435419.18, 297425.86')
-    })
-    it('switches to MGRS when this SRS is selected in the UI', () => {
-        getMousePositionAndSelect(CoordinateSystems.MGRS)
-        getMousePositionValue(100, 100, '31TFN6781597235')
-    })
-    it('switches to WebMercator when this SRS is selected in the UI', () => {
-        getMousePositionAndSelect(CoordinateSystems.WGS84)
-        getMousePositionValue(100, 100, '47° 48′ 23.71″ N 5° 14′ 28.92″ E')
-    })
-    it('goes back to LV95 display if selected again', () => {
-        // Change display projection without moving the mouse
-        getMousePositionAndSelect(CoordinateSystems.MGRS)
-        getMousePositionAndSelect(CoordinateSystems.LV95)
-        getMousePositionValue(100, 100, '2435419.18, 1297425.86')
+    context('LocationPopUp when rightclick on the map', function () {
+        it('Test the LocationPopUp is visible', () => {
+            cy.goToMapView()
+            // doing a right click
+            cy.get('[data-cy="map"]').rightclick(200, 100)
```

Comment:
here you should then test that your popup is visibile on the screen by grabbing it with `cy.get('[data-cy="..."]').should('be.visible')` or something similar